backend/src/index.js

```
@@ -1,36 +1,50 @@
 const express = require('express');
 const cors = require('cors');
+const http = require('http');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)